addOns/pscanrulesAlpha/src/test/java/org/zaproxy/zap/extension/pscanrulesAlpha/SubResourceIntegrityAttributeScannerTest.java

```diff
@@ -208,6 +208,23 @@ public void shouldIgnoreInvalidFormattedHostname() throws HttpMalformedHeaderExc
         assertThat(alertsRaised.size(), equalTo(1));
     }
 
+    @Test
+    public void shouldNotRaiseAlertGivenHtmlAssetIsInline() throws HttpMalformedHeaderException {
+        // Given
+        HttpMessage msg =
+                buildMessage(
+                        "<html><head>"
+                                + "<script src=\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABAQMAAAAl21bKAAAAA1BMVEX/TQBcNTh/AAAAAXRSTlPM0jRW/QAAAApJREFUeJxjYgAAAAYAAzY3fKgAAAAASUVORK5CYII=\"></script>"
+                                + "<link href=\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABAQMAAAAl21bKAAAAA1BMVEX/TQBcNTh/AAAAAXRSTlPM0jRW/QAAAApJREFUeJxjYgAAAAYAAzY3fKgAAAAASUVORK5CYII=\">"
```

Comment:
The Base64 value can be trimmed/removed, not meaningful for the test.

Reply:
I'll switch to `data:text/html` payloads so that they're shorter.

Reply:
It's not necessary to change the type, the contents don't affect the outcome of the test so better not have them at all.

Reply:
Since the type doesn't either (affect the outcome, that is) I can strip that out too. I think `data:,` would be the minimally correct string.

Reply:
Sounds good.